### PR TITLE
[platform/sfp]Enhance the LPM test by performing the test only on cables supporting LPM

### PR DESCRIPTION
**Pull request by stephenxs**
### Description of PR

Summary:
Fixes # (issue)
Enhance the LPM test by performing the test only on cables supporting LPM

Signed-off-by: Stephen Sun <stephens@nvidia.com>

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### CODE REVIEW

File: tests/platform_tests/test_sfp.py [hunk 1]
```diff
@@ -211,18 +211,28 @@ def test_check_sfp_low_power_mode(duthost, conn_graph_facts):
 
     logging.info("Try to change SFP lpmode")
     tested_physical_ports = set()
+    not_supporting_lpm_physical_ports = set()
     for intf in conn_graph_facts["device_conn"]:
         phy_intf = portmap[intf][0]
         if phy_intf in tested_physical_ports:
             logging.info("skip tested SFPs {} to avoid repeating operating physical interface {}".format(intf, phy_intf))
             continue
+        type = duthost.command('redis-cli -n 6 hget "TRANSCEIVER_INFO|{}" type'.format(intf))["stdout"]
```
Comment (wangxin) [inline tests/platform_tests/test_sfp.py:220]: `type` will shadow the python built-in function. Suggest to use another variable name.

**Review by wangxin — CHANGES_REQUESTED** [review]

**Comment by wangxin** [comment]
retest this please

File: tests/platform_tests/test_sfp.py (hunk 1 above)
Comment (stephenxs) [reply tests/platform_tests/test_sfp.py]: Fixed.

**Review by wangxin — APPROVED** [review]

**Comment by yxieca** [comment]
@stephenxs can you resolve the merge conflict?

**Comment by stephenxs** [comment]
> @stephenxs can you resolve the merge conflict?

Done.
Thanks

**Pull request merged**